packages.yml

```diff
@@ -0,0 +1,3 @@
+packages:
+ - package: fishtown-analytics/dbt_utils
+   version: '>=0.1.20'
```

Comment:
I don't think we need this version number to be quite so high, right? When did these two functions exist? I feel like it was many version ago...